src/tests/Directory.Build.props

```diff
@@ -53,15 +53,46 @@
    -->
   <PropertyGroup>
     <CLRTestPriorityToBuild>0</CLRTestPriorityToBuild>
+    <CLRTestPriorityToBuild Condition="'$(__Priority)' != ''">$(__Priority)</CLRTestPriorityToBuild>
+
+    <WindowsHost>$([MSBuild]::IsOSPlatform(Windows))</WindowsHost>
+
+    <TargetsWindows>false</TargetsWindows>
+    <TargetsWindows Condition="'$(TargetOS)' == 'windows'">true</TargetsWindows>
+
+    <ScriptExt>.sh</ScriptExt>
+    <ScriptExt Condition="$(WindowsHost)">.cmd</ScriptExt>
+
+    <DotNetCli>"$(RepoRoot)\dotnet$(ScriptExt)"</DotNetCli>
+
+    <XunitTestBinBase Condition="'$(XunitTestBinBase)' == ''">$(__TestBinDir)</XunitTestBinBase>
+    <XunitTestBinBase>$([MSBuild]::NormalizeDirectory('$(XunitTestBinBase)/'))</XunitTestBinBase>
+    <CORE_ROOT Condition="'$(CORE_ROOT)' == ''">$(XunitTestBinBase)Tests/Core_Root</CORE_ROOT>
+
+    <TestBuildMode Condition="'$(__TestBuildMode)' != ''">$(__TestBuildMode)</TestBuildMode>
+    <RuntimeFlavor Condition="'$(__RuntimeFlavor)' != ''">$(__RuntimeFlavor)</RuntimeFlavor>
+
+    <RestoreDefaultOptimizationDataPackage Condition="'$(RestoreDefaultOptimizationDataPackage)' == ''">false</RestoreDefaultOptimizationDataPackage>
+    <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
+
+    <UsePartialNGENOptimization Condition="'$(UsePartialNGENOptimization)' == ''">false</UsePartialNGENOptimization>
+
+    <RunWithAndroid>false</RunWithAndroid>
+    <RunWithAndroid Condition="'$(TargetOS)' == 'Android'">true</RunWithAndroid>
+
+    <MonoAot>false</MonoAot>
+    <MonoAot Condition="'$(__MonoAot)' == '1'">true</MonoAot>
+
+    <MonoFullAot>false</MonoFullAot>
+    <MonoFullAot Condition="'$(__MonoFullAot)' == '1'">true</MonoFullAot>
+
+    <MonoBinDir>$(__MonoBinDir)</MonoBinDir>
   </PropertyGroup>
 
   <!-- Which tests can we build? Default: Build managed tests for this target.
     At the command-line, the user can specify:
     +  /p:CLRTestBuildAllTargets=allTargets    Build managed tests for all target platforms.
    -->
```

Comment:
This comment looks out of place. Is it still relevant?

Reply:
Fair enough. The comment is still relevant but it's kind of orphaned after the code shuffles, I need to find a new home for it :-).